### DISCUSSION TITLE
Support for .NET 8.0.200

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#15044] [General]
.NET 8.0.200 has been released yesterday, and fixes https://github.com/dotnet/runtime/issues/96872,  https://github.com/dotnet/sdk/issues/37958 and https://github.com/dotnet/runtime/issues/93860.

To update:
- If you're on Windows and you use Visual Studio, update to Visual Studio 17.9, then [update/run uno-check](https://platform.uno/docs/articles/external/uno.check/doc/using-uno-check.html) (1.20 or higher).
- If you're using VS Code or Rider for any platform, [update/run uno-check](https://platform.uno/docs/articles/external/uno.check/doc/using-uno-check.html)  (1.20 or higher).

---
<details><summary>Previous update on 8.0.101</summary>
<p>
The new service of .NET 8.0.101 has been released, and it is causing some issues that have yet to be fixed:
- https://github.com/dotnet/runtime/issues/96872
- https://github.com/dotnet/sdk/issues/37958

Those two issues impact mobile development only, so if you're doing Windows, Skia or WebAssembly development, you will not be having issues.

uno.check does not have an update to 8.0.101 as a result of those issues.

If you're running Visual Studio 2022, you will likely want to include a [global.json](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json) in your project and force the version to 8.0.100.
</p>
</details>

**Comment by AppleTree1685** [comment]
After adding global.json below, the first issue keeps happening, does anyone else work?
```
{
  "sdk": {
    "version": "8.0.100"
  }
}
```

**Reply by jeromelaban** [reply]
If it still happens, this means workloads have not been restored, you'll need to clear all workloads (using `dotnet workload clean --all`  and re-rerun uno.check.

**Reply by jeromelaban** [reply]
An update has been posted on this thread about the release of 8.0.200 and Visual Studio 17.9, please see at the top.
